forms-shared/src/schemas/zavazneStanoviskoKInvesticnejCinnosti.ts

```diff
@@ -192,24 +197,100 @@ export default schema(
               'Čísla a názvy všetkých ostatných stavieb (ak sa jedná o súbor stavieb) vo formáte stavba 02 - Názov stavby - stavebné objekty',
           },
         ),
+        radioGroup(
+          'obsahujeByty',
+          {
+            type: 'boolean',
+            title: 'Nachádzajú sa v navrhovanej stavbe byty?',
+            required: true,
+            items: [
+              { value: true, label: 'Áno' },
+              { value: false, label: 'Nie', isDefault: true },
+            ],
+          },
+          {
+            variant: 'boxed',
+            orientations: 'row',
+          },
+        ),
+        conditionalFields(createCondition([[['obsahujeByty'], { const: true }]]), [
+          number(
+            'pocetBytovCelkovo',
+            {
+              type: 'integer',
+              title: 'Počet bytov celkovo',
+              required: true,
+              minimum: 1,
+            },
+            {},
+          ),
+          number(
+            'pocet1IzbovychBytov',
+            {
+              type: 'integer',
+              title: 'Počet 1-izbových bytov',
+              required: true,
+              minimum: 0,
+            },
+            {
+              helptext:
+                'Ak sa v navrhovanej stavbe nenachádzajú byty daného typu, uveďte v kolónke „0".',
+            },
+          ),
+          number(
+            'pocet2IzbovychBytov',
+            {
+              type: 'integer',
+              title: 'Počet 2-izbových bytov',
+              required: true,
+              minimum: 0,
+            },
+            {
+              helptext:
+                'Ak sa v navrhovanej stavbe nenachádzajú byty daného typu, uveďte v kolónke „0".',
+            },
+          ),
+          number(
+            'pocet3IzbovychBytov',
+            {
+              type: 'integer',
+              title: 'Počet 3-izbových bytov',
+              required: true,
+              minimum: 0,
+            },
+            {
+              helptext:
+                'Ak sa v navrhovanej stavbe nenachádzajú byty daného typu, uveďte v kolónke „0".',
+            },
+          ),
+          number(
+            'pocet4IzbovychBytov',
+            {
+              type: 'integer',
+              title: 'Počet 4-izbových bytov',
+              required: true,
+              minimum: 0,
+            },
+            {
+              helptext:
+                'Ak sa v navrhovanej stavbe nenachádzajú byty daného typu, uveďte v kolónke „0".',
+            },
+          ),
+          number(
+            'pocetViacAko4IzbovychBytov',
+            {
+              type: 'integer',
+              title: 'Počet viac ako 4-izbových bytov',
+              required: true,
+              minimum: 0,
+            },
+            {
+              helptext:
+                'Ak sa v navrhovanej stavbe nenachádzajú byty daného typu, uveďte v kolónke „0".',
```

Comment:
Wrong end quotation signs. Notice that in Slovak they should never be "straight" in this usage style.
Used: „0"
Correct: „0“